modules/Staff/staff_view_details.php

```diff
@@ -37,20 +37,14 @@
         echo __('The highest grouped action cannot be determined.');
         echo '</div>';
     } else {
-        $gibbonPersonID = $_GET['gibbonPersonID'];
-        if ($gibbonPersonID == false) {
+        $gibbonPersonID = $_GET['gibbonPersonID'] ?? '';
```

Comment:
Would it not be better to use null here?

Reply:
In all the code we're using '', then I continued with the pattern.

Reply:
Yep, thanks, that's my doing! 😄 I prefer an empty string for GET values rather than null. Since they're being parsed from a URL string, the empty value is already a string type. Introducing null means scripts need to be more careful to use `empty()` to check for '' and null, and some of the legacy scripts still do an ` == ''` check.